worldedit/wand.lua

```diff
@@ -1,51 +1,41 @@
 minetest.register_tool("worldedit:wand", {
 	description = "WorldEdit wand tool. Left-click to set the 1st position, Right-click to set the 2nd position.",
-	groups = {},
 	inventory_image = "worldedit_wand.png",
-	wield_image = "",
-	wield_scale = {x=1,y=1,z=1},
-	stack_max = 1, -- there is no need to have more than one
```

Comment:
wield_scale and stack_max are the default values here
l don't remember why l removed setting the wield_image to ""